azure/README.md

```diff
@@ -184,6 +184,16 @@ docker build -t azure \
 -f Dockerfile .
 ```
 
+## Build CAA Container Image
+
+If you have made changes to the Cloud API Adaptor code and if you want to deploy those changes then follow [these instructions](https://github.com/confidential-containers/cloud-api-adaptor/blob/staging/install/README.md#building-custom-cloud-api-adaptor-image) to build the container image from the root of this repository.
+
+If you would like to deploy the latest code from the default branch (`staging`) of this repository then just expose the following environment variable:
```

Comment:
Hmm isn't the default to install from `quay.io/confidential-containers` ? IOW for installing the latest code do we need to export the `registry` variable ?

Reply:
I guess the idea is to keep the following code generic, and the current code doesn't work if `registry` is unset: https://github.com/confidential-containers/cloud-api-adaptor/blob/staging/azure/README.md?plain=1#L295

Reply:
Thanks @katexochen @surajssd for the explanation